docs/bn/docs/index.md

```diff
@@ -0,0 +1,464 @@
+- **জোরালো**: স্বয়ংক্রিয় ইন্টারেক্টিভ ডকুমেন্টেশন সহ, উৎপাদনের জন্য প্রস্তুত কোড পাওয়া যাবে।
+- **মান-ভিত্তিক**: API-এর জন্য উন্মুক্ত মানগুলির উপর ভিত্তি করে (এবং সম্পূর্ণরূপে সামঞ্জস্যপূর্ণ): <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (যাহা পূর্বে Swagger নামে পরিচিত ছিল) এবং <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
+
+<small>\* এক দল প্রোডাকশন এপ্লিকেশন বানানো অভ্যন্তরীণ ডেভেলপার এর ওপর নিরীক্ষন ভিত্তিক অনুমান।</small>
+
+## স্পনসর গণ
+
+<!-- sponsors -->
+
+{% if sponsors %}
+{% for sponsor in sponsors.gold -%}
+<a href="{{ sponsor.url }}" target="_blank" title="{{ sponsor.title }}"><img src="{{ sponsor.img }}" style="border-radius:15px"></a>
+{% endfor -%}
+{%- for sponsor in sponsors.silver -%}
+<a href="{{ sponsor.url }}" target="_blank" title="{{ sponsor.title }}"><img src="{{ sponsor.img }}" style="border-radius:15px"></a>
+{% endfor %}
+{% endif %}
+
+<!-- /sponsors -->
+
+<a href="https://fastapi.tiangolo.com/fastapi-people/#sponsors" class="external-link" target="_blank">অন্যান্য স্পনসর গণ</a>
+
+## মতামত সমূহ
+
+"_[...] আমি আজকাল প্রচণ্ড পরিমানে **FastAPI** ব্যবহার করছি। [...] আমি ভাবছি মাইক্রোসফ্টে **ML সার্ভিস** এ আমার সকল দলের জন্য এটি ব্যবহার করব। যার মধ্যে কিছু পণ্য **Windows** এ সংযোযন হয় এবং কিছু **Office** এর সাথে সংযোযন হচ্ছে।_"
+
+<div style="text-align: right; margin-right: 10%;">কবির খান - <strong>মাইক্রোসফ্টে</strong> <a href="https://github.com/tiangolo/fastapi/pull/26" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_আমরা **FastAPI** লাইব্রেরি গ্রহণ করেছি একটি **REST** সার্ভার তৈরি করতে, যা **ভবিষ্যদ্বাণী** পাওয়ার জন্য কুয়েরি করা যেতে পারে। [লুডউইগের জন্য]_"
+
+<div style="text-align: right; margin-right: 10%;">পিয়েরো মোলিনো, ইয়ারোস্লাভ দুদিন, এবং সাই সুমন্থ মিরিয়ালা - <strong>উবার</strong> <a href="https://eng.uber.com/ludwig-v0-2/" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_**Netflix** আমাদের **ক্রাইসিস ম্যানেজমেন্ট** অর্কেস্ট্রেশন ফ্রেমওয়ার্ক: **ডিসপ্যাচ** এর ওপেন সোর্স রিলিজ ঘোষণা করতে পেরে আনন্দিত! [যাকিনা **FastAPI** দিয়ে নির্মিত]_"
+
+<div style="text-align: right; margin-right: 10%;">কেভিন গ্লিসন, মার্ক ভিলানোভা, ফরেস্ট মনসেন - <strong>নেটফ্লিক্স</strong> <a href="https://netflixtechblog.com/introducing-dispatch-da4b8a2a8072" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_আমি **FastAPI** নিয়ে চাঁদের সমান উৎসাহিত। এটি খুবই মজার!_"
+
+<div style="text-align: right; margin-right: 10%;">ব্রায়ান ওকেন - <strong><a href="https://pythonbytes.fm/episodes/show/123/time-to-right-the-py-wrongs?time_in_sec=855" target="_blank">পাইথন বাইটস</a> পডকাস্ট হোস্ট</strong> <a href="https://twitter.com/brianokken/status/1112220079972728832" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_সত্যিই, আপনি যা তৈরি করেছেন তা খুব মজবুত এবং পরিপূর্ন দেখায়৷ অনেক উপায়ে, আমি যা **Hug** করতে চেয়েছিলাম - কাউকে তাঁহাই তৈরি করতে দেখে আমি সত্যিই অনুপ্রানিত৷_"
```

Comment:
Can we restructure this? 

> সত্যিই, আপনি যা তৈরি করেছেন তা খুব মজবুত এবং পরিপূর্ন দেখায়৷

সত্যিই, আপনি যা তৈরি করেছেন তা খুব মজবুত এবং পরিপূর্ন ৷

> অনেক উপায়ে, আমি যা **Hug** করতে চেয়েছিলাম - কাউকে তাঁহাই তৈরি করতে দেখে আমি সত্যিই অনুপ্রানিত৷

অনেক উপায়ে, আমি যা **Hug**-এ করতে চেয়েছিলাম - তা তৈরি করতে দেখে আমি সত্যিই অনুপ্রানিত৷